### PR TITLE
Add visits join tables

### PR DESCRIPTION
**Pull request by keroloussamy**
In this PR, I made the following

- [ ]  Understand the different types of relationships between tables.
- [ ]  Prepare complex queries that answer analytical questions.
- [ ]  Use primary key & foreign key mechanisms for joining tables.

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -21,4 +21,30 @@ CREATE TABLE owners (
...
+
+CREATE TABLE visits (
+  id INT PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  vets_id INT NOT NULL,
+	animals_id INT NOT NULL,
+  date DATE NOT NULL,
+	FOREIGN KEY (vets_id) REFERENCES vets (id) ON DELETE RESTRICT ON UPDATE CASCADE,
+	FOREIGN KEY (animals_id) REFERENCES animals (id) ON DELETE RESTRICT ON UPDATE CASCADE
+);
+
```
Comment (andersonlebon) [inline schema.sql:50]: - [ ]  Kindly fix this issue I am getting when trying to create 2 tables `visits` and the `specialization` . Check here :point_down: 

image

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @keroloussamy 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## To highlight :champagne: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (keroloussamy) [reply schema.sql:50]: Hello, @andersonlebon I think the problem with the `set` word before the foreign key. But my code doesn't have it.

image

**Review by oloomoses — APPROVED** [review]
Hi @keroloussamy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**